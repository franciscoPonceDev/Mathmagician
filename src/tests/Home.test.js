import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/Home';

it('renders correct', () => {
  const tree = renderer.create(<Home />).toJSON();

  expect(tree).toMatchSnapshot();
});
