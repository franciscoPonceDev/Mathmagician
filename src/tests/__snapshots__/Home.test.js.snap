// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correct 1`] = `
<section
  className="home"
>
  <h1>
    Welcome to Mathmagician
  </h1>
  <p>
    Id enim mollit ipsum nostrud. Dolor ipsum quis laborum quis amet adipisicing dolore excepteur do elit excepteur ea. Magna occaecat consequat anim aliquip Lorem dolore non sunt dolore. Nulla incididunt ex aute reprehenderit nisi consectetur nostrud.
  </p>
  <p>
    Commodo velit Lorem voluptate commodo nulla nulla adipisicing nulla exercitation ea. Do officia fugiat tempor ullamco exercitation nulla dolor excepteur labore in elit nostrud adipisicing. Sunt sunt laboris ea ut irure labore labore occaecat labore. Est laborum sint minim tempor magna reprehenderit laboris aliqua amet. Sint pariatur anim aute Lorem aute cillum sint proident. Eu exercitation ea culpa ad irure ut consequat reprehenderit amet ullamco consequat esse. Occaecat excepteur fugiat nulla ipsum cupidatat aliqua excepteur adipisicing voluptate voluptate adipisicing non irure officia.
  </p>
</section>
`;
